src/utils/getLocationValidator.js

```diff
@@ -0,0 +1,15 @@
+export default function (allowRoot) {
```

Comment:
Since this is intended to be  part of the public utilities api, please expose through `utils/index.js` and import from `utils/`